File: Codes/Bingran Li/LinkList/83. Remove Duplicates from Sorted List.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode(int x) { val = x; }
 * }
 */
public class Solution {
    public ListNode deleteDuplicates(ListNode head) {
        
        if(head==null || head.next==null){
            return head;
        }
        ListNode currentNode = head;
        while(currentNode!=null && currentNode.next!=null){
            
            if(currentNode.next.val==currentNode.val){ // If ---> wrong......
            currentNode.next = currentNode.next.next;
            }
            else   // miss this ----> maybe i should wirte some comment...
            currentNode = currentNode.next;
        }
        return head;
        
        
    }
}

File: Codes/Yangkun Huang/Linked List/83. Remove Duplicates from Sorted List.cpp
    ListNode* deleteDuplicates(ListNode* head) {
         if(head){
             ListNode *p=head;
             while(p->next)
                p->val==p->next->val ? p->next=p->next->next : p=p->next;
         }
         return head;
    }
